xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -947,6 +982,16 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Indicates a tactile marker on floor under the buttons (or aligned with).</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="MonitoringRemoteControl" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Whether the lift is equipped with a remote operation control system</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="OtherExitDoors" type="OtherExitDoorsEnumeration" default="none" minOccurs="0">
```

Comment:
If this element and level of detail is really needed: wouldn't it be consequent to allow for multiple occurrences - e.g., to indicate that on some floors we have `opposite` doors and on some we have the same situation as on the ground floor? I would further suggest to rename `none` to `same` (`none` might be misunderstood as the clausotrophic situation of no door).
We also have the element `TroughLoader` to inform about eintry and exit on opposite sides. Could this then be removed, or should its definition be refined (e.g., entry and exit on opposite sides for the typical use case of accessing or leaving a quay)?
And why is the element called Other*Exit*Doors? They are entry doors as well as exit doors.

Reply:
Maybe we could just rename it to OtherDoors for clarity

Reply:
In the Netherlands we inventoried all liftequipment at railway stations. Of a total of 422 lifts, 266 have an exit door on the opposite side (Througloader), 151 have entrance and exit on the same side and 5 lifts have en exit on the left or right. Using the througloader attribute in LiftEquipment is sufficient for us in combination with a notice for the 5 lifts with a left/right exit. We have inventoried the CompassOctant (CompassOctantEnumeration) of the lift entrance/door for all railway stations.  Instead of the OtherExitDoors the adding of CompassOctant to EquipmentPlace or LiftCallEquipment can make sense.